app/src/main/java/io/gnosis/safe/ui/safe/send_funds/SendAssetReviewViewModel.kt

```diff
@@ -251,10 +251,10 @@ class SendAssetReviewViewModel
                     SendAssetReviewState(
                         ViewAction.NavigateTo(
                             SendAssetReviewFragmentDirections.actionSendAssetReviewFragmentToSendAssetSuccessFragment(
-                                activeSafe.chain,
-                                safeTxHash,
-                                amountString,
-                                selectedAsset.symbol
+                                chain = activeSafe.chain,
+                                txId = safeTxHash, // to get non cached updates later we need the txId not the hash
```

Comment:
I got confused by the naming here, txId and safeTxHash

Reply:
Are we sure that it's caching problem? why we don't have such on iOS ?

Reply:
because ios uses the txId: 

txIds look likes this: `multisig_0xfcFab1Eb61d3b1E8ff8F1eF1CD8e46F8be30b77f_0xc958935d43d24337ff7a425445ad05ca579231edf98eab9dd008db8447751f7d`

txHashes look like this: `0xc958935d43d24337ff7a425445ad05ca579231edf98eab9dd008db8447751f7d`

Reply:
But i didn't look into ios regarding the first time txId should be passed. In Android we do not know the txId at the time. It is returned in the txDetails.

Reply:
Here is an example request: https://safe-client.staging.5afe.dev/v1/chains/5/transactions/9f157207875105636c594bcce3ebea3eb5e0704f00cf64f59e3d286bc4258189

Reply:
I'm actually not sure about this change if it's correct or not,